Jenkinsfile

```
@@ -0,0 +1,109 @@
+def testFileName = 'go-test.sh'
+
+def testFileContent = """#!/bin/sh
+${operatorName}-test -test.parallel=1 -test.failfast -root=/ -kubeconfig=incluster -namespacedMan=namespaced.yaml -test.v
+"""
+
+def dockerfileContent = """
+FROM alpine:3.6
+USER nobody
+ADD templates/*.yaml /usr/local/bin/templates/
+ADD ${operatorName} /usr/local/bin/${operatorName}
+ADD ${operatorName}-test /usr/local/bin/${operatorName}-test
+ADD deploy/operator.yaml /namespaced.yaml
+ADD go-test.sh /go-test.sh
+"""
+
+node ('operator-sdk') {
+    stage ('Checkout') {
+        checkout scm
+    }
+    stage('Vendor the dependencies') {
+        sh 'dep ensure'
+    }
+
+    openshift.withCluster('operators-test-cluster') {
+        generateKubeConfig()
+
+        stage('New project in OpenShift') {
+            openshift.newProject(openshiftProjectName)
+        }
+
+        openshift.withProject(openshiftProjectName) {
+            stage("Build ${operatorName} & ${operatorName}-test binaries") {
+                sh """
+                export GOOS=linux GOARCH=amd64 CGO_ENABLED=0
+                go build -o ${operatorName} cmd/manager/main.go
+                go test -c -o ${operatorName}-test ./test/e2e/...
+                """
+            }
+
+            stage("Create test file ${testFileName}") {
+                writeFile file: testFileName, text: testFileContent
+                sh "chmod +x ${testFileName}"
+            }
+
+            stage("Create a Dockerfile for ${operatorName}-test") {
+                writeFile file: "Dockerfile", text: "${dockerfileContent}"
+            }
+
+            stage("Modify operator image name in operator deployment file") {
+                sh "yq w -i deploy/operator.yaml spec.template.spec.containers[0].image ${operatorDockerImageName}"
+            }
+
+            stage("Create necessary resources") {
+                sh """
+                kubectl apply -f deploy/crds/crd.yaml -n ${openshiftProjectName} || true
```

Why do we need to ignore errors here?